rascaline/src/calculators/soap/power_spectrum.rs

```diff
@@ -375,6 +386,8 @@ impl SoapPowerSpectrum {
 struct SpxPropertiesToCombine<'a> {
     /// value of l
     spherical_harmonics_l: usize,
+    /// normalization factor (-1)^l * sqrt(2 l + 1)
```

Comment:
use the power of LateX
```suggestion
    /// normalization factor :math:`-1^l  \sqrt{2 l + 1}`
```